vms/platformvm/config/dynamic_fees_config.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package config
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/snow"
+	"github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/ava-labs/avalanchego/utils/units"
+
+	commonfees "github.com/ava-labs/avalanchego/vms/components/fees"
+)
+
+// eUpgradeDynamicFeesConfig to be tuned TODO ABENEGIA
+var (
+	eUpgradeDynamicFeesConfig = commonfees.DynamicFeesConfig{
+		FeeRate: commonfees.Dimensions{
+			1 * units.NanoAvax,
+			2 * units.NanoAvax,
+			3 * units.NanoAvax,
+			4 * units.NanoAvax,
+		},
+
+		BlockMaxComplexity: commonfees.Max,
+	}
+
+	preEUpgradeDynamicFeesConfig = commonfees.DynamicFeesConfig{
+		FeeRate:            commonfees.Empty,
+		BlockMaxComplexity: commonfees.Max,
+	}
+
+	customDynamicFeesConfig *commonfees.DynamicFeesConfig
+)
+
+func GetDynamicFeesConfig(isEActive bool) commonfees.DynamicFeesConfig {
+	if !isEActive {
+		return preEUpgradeDynamicFeesConfig
+	}
+
+	if customDynamicFeesConfig != nil {
+		return *customDynamicFeesConfig
+	}
+	return eUpgradeDynamicFeesConfig
+}
+
+func ResetDynamicFeesConfig(ctx *snow.Context, customFeesConfig *commonfees.DynamicFeesConfig) error {
```

Comment:
we need to provide a mechanism to override fee configuration, so that we use ad-hoc values in tests